tailwind.config.js

```diff
@@ -48,14 +49,13 @@ module.exports = {
           verticalAlign: "middle",
         },
       };
-      addVariant("summary-open", ["details[open] > summary > &"]);
-      addVariant("details-open", ["details[open] > &"]);
```

Comment:
Was this dropped on purpose?

Reply:
It was not being used so, it is being removed